### DISCUSSION TITLE
Элемент не определяется как DOM

### DISCUSSION THREAD

**Discussion by comrade-che** [open-tutorials/cypress#27] [Q&A] [answered]
Добрый день!

Не срабатывает нажатие на кнопку в интерфейсе системы. Даже уловка типа {force: true} не помогает. Подскажите, что нужно потребовать от разработчика? Скрины DOM, ошибки и куска теста прилагаю

![1](https://user-images.githubusercontent.com/118734363/216278637-861b5414-936d-4adc-972c-b05c1180b12f.png)

![2](https://user-images.githubusercontent.com/118734363/216278674-d4e6f576-545b-41a0-ba68-e053e63261c3.png)

![3](https://user-images.githubusercontent.com/118734363/216278705-1a880fb4-03b1-49a8-b617-1054148dea9f.png)

**Comment by Vladimir-Tarasov** [comment, marked as answer]
@comrade-che В данном случае `cy.wait(500)` является достаточным временем для загрузки данного элемента?

**Reply by breslavsky** [reply]
@comrade-che ошибка синтаксиса, должно быть
```js
cy.get('selector').click();
```
У тебя в коде

```js
cy.get('selector');
cy.wait(500).click();
```
